src/database/mongo/schemas/ReminderSchema.js

```diff
@@ -0,0 +1,10 @@
+const { Schema } = require('mongoose')
+
+module.exports = new Schema({
+  _id: String,
```

Comment:
_id prop is unnecessary, default in any mongoose schema

Reply:
it's useful, since _id equals the user's Discord ID

Reply:
oh, on reminders...

Reply:
nvm go ahead